lib/Model/Message.php

```diff
@@ -222,6 +223,25 @@ public function getLocalAttachments(): array {
 		return $this->localAttachments;
 	}
 
+	/**
+	 * @param Attachment $attachment
+	 *
+	 * @return void
+	 *
+	 * Adds a file that's coming from another email's attachment (typical
+	 * use case is forwarding a message)
+	 *
+	 */
+	public function addAttachmentFromAttachment(Attachment $attachment) {
+		$part = new Horde_Mime_Part();
+		$part->setCharset('us-ascii');
```

Comment:
this sounds like a assumption. do we know if this is always correct?

Reply:
Well, I've just copied if from addAttachmentFromFiles and addLocalAttachment 🙈

Reply:
Well then yolo on :speak_no_evil: